src/sentry/tasks/post_process.py

```diff
@@ -44,6 +44,43 @@ def _get_service_hooks(project_id):
     return result
 
 
+def _should_send_error_created_hooks(project):
+    from sentry.models import ServiceHook, Organization
+    from sentry import options
+    import random
+
+    cache_key = u'servicehooks-error-created:1:{}'.format(project.id)
+    result = cache.get(cache_key)
+
+    if result is None:
+
+        use_sampling = options.get('post-process.use-error-hook-sampling')
+        org = Organization.objects.get_from_cache(id=project.organization_id)
```

Comment:
If you are using sampling you are not using this. You may avoid fetching the organization in that case. Unless you also want to check the feature when the sampling is on, then the check is missing below.

Reply:
> You may avoid fetching the organization in that case

will move that into the else

> Unless you also want to check the feature when the sampling is on, then the check is missing below.

I don't think we would need both